src/components/MoneyReportHeader.js

```diff
@@ -141,43 +150,50 @@ function MoneyReportHeader({session, personalDetails, policy, chatReport, report
                     </View>
                 )}
             </HeaderWithBackButton>
-            {shouldShowSettlementButton && isSmallScreenWidth && (
-                <View style={[styles.ph5, styles.pb2, isSmallScreenWidth && styles.borderBottom]}>
-                    <SettlementButton
-                        currency={moneyRequestReport.currency}
-                        policyID={moneyRequestReport.policyID}
-                        chatReportID={moneyRequestReport.chatReportID}
-                        iouReport={moneyRequestReport}
-                        onPress={(paymentType) => IOU.payMoneyRequest(paymentType, chatReport, moneyRequestReport)}
-                        enablePaymentsRoute={ROUTES.ENABLE_PAYMENTS}
-                        addBankAccountRoute={bankAccountRoute}
-                        shouldShowPaymentOptions
-                        formattedAmount={formattedAmount}
-                    />
-                </View>
-            )}
-            {shouldShowApproveButton && isSmallScreenWidth && (
-                <View style={[styles.ph5, styles.pb2, isSmallScreenWidth && styles.borderBottom]}>
-                    <Button
-                        success
-                        medium
-                        text={translate('iou.approve')}
-                        style={[styles.w100, styles.pr0]}
-                        onPress={() => IOU.approveMoneyRequest(moneyRequestReport)}
-                    />
-                </View>
-            )}
-            {shouldShowSubmitButton && isSmallScreenWidth && (
-                <View style={[styles.ph5, styles.pb2, isSmallScreenWidth && styles.borderBottom]}>
-                    <Button
-                        medium
-                        success={chatReport.isOwnPolicyExpenseChat}
-                        text={translate('common.submit')}
-                        style={[styles.w100, styles.pr0]}
-                        onPress={() => IOU.submitReport(moneyRequestReport)}
-                    />
-                </View>
-            )}
+            <View style={[styles.dFlex, styles.flexColumn, styles.borderBottom]}>
```

Comment:
One last nitpick 🤓 @koko57 This leads to double bottom border if there is no next steps:

<img width="806" alt="image" src="https://github.com/Expensify/App/assets/36083550/4200d815-7eaa-42ad-abee-664f58fa3760">

<img width="794" alt="image" src="https://github.com/Expensify/App/assets/36083550/cbb8c795-7178-4771-b316-005847c9f8ce">